Sources/CodableTransform.swift

```diff
@@ -37,24 +37,21 @@ open class CodableTransform<T: Codable>: TransformType {
     public init() {}
 
     open func transformFromJSON(_ value: Any?) -> Object? {
-				var _data: Data? = nil
-				switch value {
-				case let dict as [String : Any]:
-					_data = try? JSONSerialization.data(withJSONObject: dict, options: [])
-				case let array as [[String : Any]]:
-					_data = try? JSONSerialization.data(withJSONObject: array, options: [])
-				default:
-					_data = nil
-				}
-				guard let data = _data else { return nil }
-				
-        do {
-            let decoder = JSONDecoder()
-            let item = try decoder.decode(T.self, from: data)
-            return item
-        } catch {
-            return nil
-        }
+		guard let item = value else {
+			return nil
+		}
+		
+		if JSONSerialization.isValidJSONObject(item) {
+			guard let data = try? JSONSerialization.data(withJSONObject: item, options: []) else {
+				return nil
+			}
+			return try? JSONDecoder().decode(T.self, from: data)
+		} else {
+			guard let data = try? JSONSerialization.data(withJSONObject: ["value": item], options: []) else {
```

Comment:
I wrote `Single Value` parsing code by referring to [Moya](https://github.com/Moya/Moya/)'s code.
Link: [Moya/Response.swift/Line-167](https://github.com/Moya/Moya/blob/218c2f3462ff1e8b85283b94829afd54a9850664/Sources/Moya/Response.swift#L167)